src/Microsoft.AspNetCore.Razor.Evolution/Intermediate/CSharpAttributeValueIRNode.cs

```diff
@@ -9,16 +9,14 @@ namespace Microsoft.AspNetCore.Razor.Evolution.Intermediate
 {
     public class CSharpAttributeValueIRNode : RazorIRNode
     {
-        public override IList<RazorIRNode> Children { get; } = EmptyArray;
+        public override IList<RazorIRNode> Children { get; } = new List<RazorIRNode>();
```

Comment:
Would it make sense to do this in the base type and change this to a lazy List if we find that we're creating too many of these? Seems super repetitive to have ` = new List<RazorIRNode>();` everywhere.